app/views/directory_objects/personPlacard.html.erb

```diff
@@ -1,46 +1,156 @@
+					toggleBGImg = 1;
+					break;
+				  } 
+			}
+		function changeInfo(newName)  
+			{
+			var entTitle = document.getElementById("newTitle").value;
+			 document.getElementById("titleField").innerHTML= entTitle;
+
+			//This parses data to insert in optional extra info field
+			var entInfo = document.getElementById("newInfo").value;
+			var infoFormat = entInfo.replace(/\n/g, "<br>"); //Change newline encoding
+
+			//Count number of lines in extra info, if too many then display error
+			var numLines = (entInfo.match(/\n/g) || []).length;
+
+			if(numLines < 5){
+				document.getElementById("extraField").innerHTML= infoFormat;
+				document.getElementById("errorField").innerHTML= "";
+			}
+			else{
+				document.getElementById("errorField").innerHTML= "ERROR: too many lines";
+				document.getElementById("extraField").innerHTML= "";
+			}
+
+			}
+			
+		</script>
+
+		<!-- This is the placard background -->
 		<div class="backgroundImg">
-      <%= image_tag("placard/back1.jpg", id: "bgImage", alt: "back", class: "bgstyle") %>
+      		<%= image_tag("placard/back1.jpg", id: "bgImage", alt: "", class: "bgstyle") %>
 
 			<!-- This is the name field -->
 			<div id="nameField" class="name"> <%= @name %>
 
-			<!--Extra (watch for div enclosure-->
-			<div id="titleField" class="title"><%= @title %></div>
+			<!--Extra (watch for div enclosure)-->
+			<div id="titleField" class="title"><%= @title %> </div>
+			<div id="extraField" class="extrainfo"></div>
 
 			</div>
-
 			<!-- This is the QR image-->
 			<div class="qr">
-          <%= image_tag(@qrLink, alt: "qrCode", class: "qrstyle", id: "qrImage") %>
+          		<%= image_tag(@qrLink, alt: "qrCode", class: "qrstyle", id: "qrImage") %>
 			</div>
 
 			<!-- ===============EXTRA=================-->
+			<!-- This is the school logo -->
 			<div class="logo">
-        <!-- image_tag("icon.png", size: "16x10", alt: "Edit Entry") -->
-					<%= image_tag("placard/davislogo.png", alt: "logoCode", class: "logostyle", id: "logoImage") %>
+			<%= image_tag("placard/davislogo.png", alt: "logoCode", class: "logostyle", id: "logoImage") %>
 			</div>
 
+			<!-- This is the department name-->
+			<div id="depNameField" class="depname">
+			<%= @department %>
+			</div>
 
-			<!-- This is the department name field -->
-			<div id="depNameField" class="depname"><%= @department %></div>
-
+			<!-- This is the email address-->
+			<div id="emailField" class="email">
+			<%= @email %>
+			</div>
 			<!-- =====================================-->
 		</div>
 
-		<p> Cut on dotted line </p>
-		<p> Thanks for participating in the new and improved wayfinding system! </p>
+		<!-- This is where the user can change the background and add a title or additional info-->
+		<br>
+		<button onclick="changeBgImage()">Change Background Image</button>
+
+		<br>
+		<h4>
+		Optional fields
+		</h4>
+		<input type="text" id="newTitle" size="35" placeholder="Enter your title"> 
```

Comment:
It would be best if office hours, phone number, etc. were properly fields in the database but I think it would be unreasonable to ask you to dive into Rails + Angular at this hour. We can talk about doing a code sprint to do that during our next meeting but I'm fine leaving it as is.